README.md

```
@@ -24,10 +24,11 @@ If `gh auth token` fails or you get a 401, re-authenticate with the `read:packag
 gh auth login --scopes read:packages
 ```
 
-**2. Install** — add the registry to `.npmrc`, then install:
+**2. Install** — add both lines to `.npmrc`, then install (after `josh init` these are managed automatically):
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)